packages/system/src/theme.ts

```diff
@@ -28,30 +28,43 @@ export type ScaleValue<T> = T | T[] | NestedScaleDict<T> | undefined;
  * descriptive name for the scale (e.g. `small`/`medium`/.. or `body`/`heading`/...),
  * and the value is the CSS value.
  */
-export type Scale<T> = {
+export interface Scale<T> {
   [key: string]: ScaleValue<T>;
-};
+}
+
+/**
+ * A {@link Scale} that also includes a required `none` value, which is
+ * usually used to define the blank value (e.g `0`).
+ */
+export interface ZeroScale<T> extends Scale<T> {
```

Comment:
Why not putting the two zeroScales to the other SizeScales?

Reply:
Not sure what you mean 🤔

Reply:
Why we cant combine Scale and ZeroScale together in one interface?
Or SizeScale with zeroSizeScale. E.g: add none to SizeScale without having an extra ZeroSizeScale

Reply:
ZeroScale and Scale: because we have properties that don't have a "zero", same for SizeScale and ZeroSizeScale. it depends how strickt we want to be with our `Theme` definition.